File: 8-kyu/001-even-or-odd/001-even-or-odd.py
def even_or_odd(number):
    if number % 2 == 0:
        return 'Even'
    else:
        return 'Odd'


def even_or_odd(number):
    return "Even" if number % 2 == 0 else "Odd"


def even_or_odd(number):
    return ["Even", "Odd"][number % 2]


print(even_or_odd(89))

File: 8-kyu/001-even-or-odd/001-even-or-odd.swift
func evenOrOdd(_ number:Int) -> String {
  if( number % 2 == 0) {
      return "Even"
  } else {
      return "Odd"
  }
}

func evenOrOdd(_ number:Int) -> String {
    return (number % 2 == 0) ? "Even" : "Odd"
}

func evenOrOdd(_ number:Int) -> String {
    return number.isMultiple(of: 2) ? "Even" : "Odd"
}

func evenOrOdd(_ number:Int) -> String {
    return ["Even", "Odd"][number % 2]
}

func evenOrOdd(_ number:Int) -> String {
    return number.isEven()
}

extension  Int {
    func isEven() -> String {
        return self.isMultiple(of: 2) ? "Even" : "Odd"
    }
}

print(evenOrOdd(12))